app/controllers/bookings_controller.rb

```diff
@@ -61,14 +85,32 @@ def destroy
     end
   end
 
+  def get_city_wise_cleaner
+    @cleaner_temp = CitiesCleaner.where(city_id:City.find_by(name:params[:city]))
```

Comment:
@mayur-kambariya the subquery should only load the `id` column from `City` model !